packages/cli/src/platform/accelerate/disable.ts

```diff
@@ -26,6 +26,6 @@ export class Disable implements Command {
         intent: 'disable',
         projectId: project,
       },
-    })
+    }) as Promise<any>
```

Comment:
These have been added to make what was previously implicitly this type explicit.

It needs to be fixed later.

Right now the local CLI framework here only allows strings to be returned.

Changing this requires increasing the scope of this PR and I think we can defer that.

The casts here are not changing behaviour, so, its still progress, by making them explicit, in this PR.

Reply:
Let's add some of your remarks as TODO comments?

Reply:
There's a bunch of callsites though and in this case the `as` casting serves as kind of todo anyways. But feel free to push a commit if that helps!